test/verify/check-system-info

```diff
@@ -107,6 +107,10 @@ class TestSystemInfo(testlib.MachineCase):
         # need to wait until CPU usage settles down, to avoid a layout-shifting error/warning icon
         b.wait_not_present("#system-usage-cpu-progress + td .pf-v5-c-progress__status-icon")
         testlib.wait(lambda: b.get_pf_progress_value("#system-usage-cpu-progress + td") < 30)
+        # Wait for tuned text to appear, even thought we ignore it we need to button to have some text.
+        if b.pixels_label:
+            recommended_profile = m.execute("tuned-adm recommend").strip()
```

Comment:
Ah, we do that in check-system-tuned, but not in -info. It could not hurt to just do this unconditionally, for all test runs, but not important.

Reply:
Right, and the universe is clearly messing with me, because we got another pixel failure

https://cockpit-logs.us-east-1.linodeobjects.com/pull-21584-6bcc1e1a-20250204-110657-fedora-41-expensive/log.html

Reply:
It was a fun idea to check all systems but then we'd need to stop tuned everywhere so we get a consistent expected text. As I get this flake in a few PR's now, let's merge it with a conditional for pixel tests

Reply:
Agreed. Sorry, I wasn't aware it caused so much issues.